Syndicate uniform supplies

heepox opened the pull request:
For when you want to look stylishly evil.

## About The Pull Request

Adds a Syndicate Uniform Supplies Crate Containing some Stylish Syndicate Gear
Tactical Turtlenecks
Combat Boots
Syndicate Berets
Armored Vests (no syndicate Equivalent sadly)
Combat Gloves
Balaclavas (Still cool might change to Syndie masks but i dont want them to look totally like nukies)

## Why It's Good For The Game

Adds the ability for someone to order Syndicate Clothes and some armor from an Emagged Cargo console for 3000 credits to equip themselves and 2 others in Fashionably evil clothing.

## Changelog
:cl:
add: Syndicate Uniform Supplies crate to EMAG:ed cargo console containing three sets of syndicate clothing and equipment to wear. Costs 3000.
/:cl:

heepox committed:
Merge pull request #3 from BeeStation/master

Pulling to my fork hope i did it right

heepox committed:
Merge pull request #5 from BeeStation/master

pull

heepox committed:
Merge pull request #7 from BeeStation/master

pulling to my own fork

heepox committed:
Merge branch 'BeeStation:master' into master

heepox committed:
Syndicate Uniform Supplies

For when you want to look stylishly evil.

ItsTisme commented:
Id make them more expensive purely because of combat gloves being insulated. Other then that looks fun

BrumalLake commented:
the contraband pack and smugglers already have syndicate masks
save the people a few bucks will you

williamcll commented:
Can you put this on the nukie outpost as well?

Boxlta commented:
add syndicate admirial equipment to a second crate for like, 5000

williamcll commented:
> add syndicate admirial equipment to a second crate for like, 5000

Pretty sure it's called strategist.

Howluinb commented:
Do I misunderstand what you mean by Armored Vests, or do you mean stuff like Armored greatcoat/overcoat ect ?
Those are literally way too strong to be buyable

BrumalLake commented:
armored vest as in mk IV swat hardsuit
yes, armored vest exactly as written in the code
aand you don't even need an electromagnetic sequencer to buy that stuff anyways

heepox commented:
> armored vest as in mk IV swat hardsuit
> yes, armored vest exactly as written in the code
> aand you don't even need an electromagnetic sequencer to buy that stuff anyways

yes its the ones sec and barman have. Literally called that in code. Its there for convenience. and you dont need an emag to buy it but you need sec access to open if you dont have emag.

Archanial commented on code/modules/cargo/packs.dm:
```diff
@@ -274,6 +274,32 @@
 
+/datum/supply_pack/emergency/syndieclothes
+	name = "Syndicate Uniform Supplies"
+	desc = "(*!&@#SAD ABOUT THAT NULL_ENTRY, HUH OPERATIVE? WELL, THIS LITTLE ORDER WILL MAKE YOU STYLISH SYNDICATE STYLE. CONTAINS A COLLECTION OF THREE TACTICAL TURTLENECKS, THREE COMBAT BOOTS, THREE COMBAT GLOVES, THREE BALACLAVAS, THREE SYNDICATE BERETS AND THREE ARMOR VESTS!#@*$"
```
don't mention OOC stuff IC

heepox replied:
literally copied from the other emag crate.

Archanial replied:
This doesn't justify anything. Just delete SAD NULL_ENTRY

Archanial commented on code/modules/cargo/packs.dm:
```diff
@@ -274,6 +274,32 @@
+	name = "Syndicate Uniform Supplies"
+	desc = "(*!&@#SAD ABOUT THAT NULL_ENTRY, HUH OPERATIVE? WELL, THIS LITTLE ORDER WILL MAKE YOU STYLISH SYNDICATE STYLE. CONTAINS A COLLECTION OF THREE TACTICAL TURTLENECKS, THREE COMBAT BOOTS, THREE COMBAT GLOVES, THREE BALACLAVAS, THREE SYNDICATE BERETS AND THREE ARMOR VESTS!#@*$"
+	hidden = TRUE
+	cost = 1800
```
1800 is too low

Archanial replied:
this is still a contraband and is significantly harder to ship to the station. If anything it should cost more to sponsor syndicate.

heepox replied:
other crate has actyally useful stuff... if anything this should be cheaper.

Archanial replied:
Those are illegally shipped syndicate goods, it doesn't matter if they're useful, they're hard to ship.

Archanial replied:
```suggestion
	cost = 3000
```

heepox committed:
Update code/modules/cargo/packs.dm

Cost to 3k... even though this shit is less useful than the other Emag Crate.

Co-authored-by: Archanial <37456678+Archanial@users.noreply.github.com>

heepox committed:
Update packs.dm

Deleted Reference to the removed Null crates from Spec ops and Uniform supplies crate.
